app/src/main/res/values-uk/strings.xml

```diff
@@ -32,7 +32,7 @@
     <string name="fixing">Виправлення…</string>
     <string name="dates_fixed_successfully">Дати успішно виправлені</string>
     <string name="share_resized">Поділитися зображенням іншого розміру</string>
-    <string name="upgraded_from_free">Йой,\n\nздається, ви оновились із застарілого безкоштовного додатка. Тепер ви можете видалити стару версію, у якій є кнопка \"Перейти на Pro\" вгорі налаштувань додатка.\n\nВи втратите лише елементи з Кошика, позначки улюблених елементів, а також потрібно буде скинути ваші налаштування додатка.\n\nДякую!</string>
+    <string name="upgraded_from_free">Йой,\n\nздається, ви перейшли зі старого безкоштовного додатку на цей. Тепер ви можете видалити стару версію, у якій є кнопка \"Перейти на Pro\" вгорі налаштувань додатку.\n\nВи втратите лише елементи з Кошика, позначки улюблених елементів, а також потрібно буде скинути ваші налаштування додатку.\n\nДякую!</string>
```

Comment:
Wow, you made a great job. I checked your links, reviewed several more vocabularies and have to admit that in most cases you are perfectly right.
So I'd like to offer this translation for string No.35:

> Агов, здається, ви оновились зі старого безкоштовного додатка. _(and so on)_

Reply:
Looks fine to me. But we probably have to use 'оновили' instead of 'оновились'. I just realized that the word refers to a person but not to an app.
So I suggest 
_Агов, здається, ви оновили старий безкоштовний додаток._ (and so on)